test/mongodb.test.js

```diff
@@ -1611,6 +1621,74 @@ describe('mongodb connector', function() {
       });
     });
 
+    it.only('find should be able to query around location in deep/multiple keys', function(done) {
+      var coords = { lat: 1.25, lng: 20.20 };
+
+      geoDb.autoupdate(function(err) {
+        var heroNb = 0;
+        var powers = ['fly', 'lasers', 'strength', 'drink'];
```

Comment:
They help make sure every created Superhero is different, this way I make sure there is only one matching the query below.